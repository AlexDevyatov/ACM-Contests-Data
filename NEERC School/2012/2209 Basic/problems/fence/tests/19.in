6
-37 -37
22 -32
40 -19
35 40
-18 39
-37 38
-12 18
