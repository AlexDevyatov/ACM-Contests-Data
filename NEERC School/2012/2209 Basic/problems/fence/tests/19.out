7
-37 -37
22 -32
40 -19
35 40
-18 39
-12 18
-37 38
